src/vsr.zig

```diff
@@ -1656,11 +1656,11 @@ pub const Headers = struct {
     pub const ViewChangeArray = ViewChangeHeadersArray;
 
     fn dvc_blank(op: u64) Header {
-        return std.mem.zeroInit(Header, .{
-            .command = .reserved,
-            .op = op,
-            .checksum = 0,
-        });
+        var header: Header = undefined;
+        @memset(std.mem.asBytes(&header), 0);
```

Comment:
What’s the difference between `@memset` and `std.mem.zero` (note: zero, *not* zeroInit)?

if the latter gives us extra compile-time checks that zeroing makes sense, that might be preferable.


Although I am hesitant about relying on std functions with subtle and shifting semantics.

memset certainly is simpler!

Reply:
Truthfully I forgot that `std.mem.zeroes` exists when I was writing this code. :wink: 

It looks like `std.mem.zeroes` would do what we want in this case. That said, considering this change is due to being bitten by a quiet stdlib change, I think being explicit is best.